search/lib/api/stac.js

```diff
@@ -53,6 +53,9 @@ async function getCatalog (request, response) {
   const catalog = createDefaultCatalog(request.apiGateway.event);
 
   cmrCollections.forEach((item) => {
+    if (typeof item.href === 'string' && item.href.includes('providers')) {
```

Comment:
When you run this, does it pick up the self link and try to run addNext() or does it only pick up the child links that contains 'providers'?